src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,62 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import React from 'react';
+import { Tab } from '../../types';
+
+type Props = {
+  tabs: Tab[]
+  selectedTabId: string
+  onTabSelected: (tab: Tab) => void
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(({ id }) => (
+    id === selectedTabId
+  )) || tabs[0];
+
+  const { content } = selectedTab;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const { id, title } = tab;
+
+            return (
```

Comment:
could be a bit shorter
```suggestion
          {tabs.map(({ id, title }) => (
```

Reply:
That's what I did at first, but I needed `tab` to use in `onClick`